Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN bash /tmp/env-config.sh
 # dependencies which end up bloating the image.
 # /apps/notifications is not removed because it is required by Calypso
 COPY . /calypso/
-RUN yarn install --frozen-lockfile
+RUN yarn install --immutable --check-cache
```

Comment:
You shouldn't need `--check-cache` unless you're doing zero-installs where people commit the cache. Also of note, if you set `enableGlobalCache: false` in your `.yarnrc.yml` file you can copy in the cache to speed up this step. I use this dockerignore to only copy the necessary parts from the `.yarn` folder
```gitignore
# Yarn 2
.yarn/*
!.yarn/cache
!.yarn/releases
!.yarn/plugins
!.yarn/patches
.pnp.*
```